classes/privacy/provider.php

```diff
@@ -36,7 +36,7 @@ class provider implements null_provider {
      *
      * @return  string
      */
-    public static function _get_reason() {
+    public static function get_reason(): string {
```

Comment:
this looks like a conversion from using the old legacy polyfill, but the use legacy_polyfill is still in place?

Reply:
Ah yep, good pickup. Wasn't aware of the polyfill, and have removed since its no longer necessary.